Resources/views/CRUD/list_array.html.twig

```diff
@@ -9,10 +9,10 @@ file that was distributed with this source code.
 
 #}
 {% import _self as list %}
-{%  macro render_array(value) %}
+{%  macro render_array(value, list) %}
     {% for key, val in value %}
         {% if val is iterable %}
-            [{{ key }} => {{ list.render_array(val) }}}]
+            [{{ key }} => {{ list.render_array(val, list) }}]
```

Comment:
For anyone wondering (like me), this is because twig since 2.x (I think this works with 1.x) does not support using a macro without importing it on the same block code.

Am I wrong? It would be the same doing this:

`_self.render_array....` right? Maybe it is a better solution?